src/components/BuySellForm/BuySellData/BuySellData.tsx

```diff
@@ -55,7 +60,33 @@ const BuySellData = forwardRef<HTMLDivElement, TBuySellDataProps>(
                         <Text size={valueSize}>{name}</Text>
                     </div>
                     <div className='flex flex-col w-full'>
-                        <Text color='less-prominent' size={labelSize}>{`Rate (1 ${accountCurrency})`}</Text>
+                        <div className={clsx({ 'lg:hover:mb-[2.2rem]': isFloating })}>
+                            <div
+                                className={clsx('flex items-center gap-2', {
+                                    'lg:hover:absolute': isFloating,
+                                })}
+                            >
```

Comment:
i need two divs here, one to make sure the div floats and the other to control the margin, or else it looks kinda glitchy if i do it in one